app/models/court_application_prosecution_case_summary.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CourtApplicationProsecutionCaseSummary
+  include ActiveModel::Model
+
+  attr_reader :prosecution_case
+  attr_accessor :prosecution_case_id
```

Comment:
just checking, should this be only `attr_reader` ?

Reply:
Used to initiate by other classes